sdk/chan.go

```diff
@@ -17,16 +17,16 @@ type Channel struct {
 
 // Publish : Publishes a message with the given body on the current channel
```

Comment:
Would you mind changing the commenting style into typical Go style? See https://golang.org/doc/effective_go.html#commentary. It should be

```
// Publish promulgates a message with the given body on the current channel.
```